addon/-private/modifier.ts

```diff
@@ -544,20 +542,16 @@ export default class ElementQueryModifier extends Modifier<ModifierArgs> {
       : sizeObjectsSortedAsc[sizeObjectsSortedAsc.length - 1];
   }
 
-  _maybeRunPromiseResolveHasBeenInstalled(): void {
-    if (this._promiseResolveHasBeenInstalled) {
-      this._promiseResolveHasBeenInstalled();
-      this._promiseResolveHasBeenInstalled = undefined;
-    }
+  __didResizeHandler(): void {
```

Comment:
I had to look a couple of times to realize this hasn't the same name like the other method. Maybe call this `_didResizeHandlerSync` instead?